modules/effects/arrow-effects/src/main/kotlin/arrow/effects/internal/CancelUtils.kt

```diff
@@ -0,0 +1,59 @@
+package arrow.effects.internal
+
+import arrow.effects.ForIO
+import arrow.effects.IO
+import arrow.effects.IOFrame
+import arrow.effects.fix
+import com.sun.tools.javac.util.ListBuffer
```

Comment:
We shouldn't depend on sun tools.

Reply:
Removed on later pushes, sorry, I had some code pending to push